cr8/run_crate.py

```diff
@@ -295,6 +297,9 @@ def start(self):
 
     def _set_addr(self, protocol, addr):
         log.info('{0:10}: {1}'.format(protocol.capitalize(), addr))
+        host, port = addr.rsplit(':', 1)
+        port = int(port)
+        self.addresses[protocol] = Address(host, port) 
         if protocol == 'http':
             self.http_host = addr
```

Comment:
@mfussenegger is there a way to mark this as deprecate? or will it be in the release notes for the next version?

Reply:
I'll take care of this later